### PR TITLE
Added new paper

### PR DESCRIPTION
**Pull request by wzhi**
Added paper "Unifying Scene Representation and Hand-Eye Calibration with 3D Foundation Models".

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ A curated list of papers and open-source resources related to DUSt3R, an emergin
 
 <br>
```
Comment (ruili3) [inline README.md]: you can remove the "\<br\>" tag between subfields [Gaussian Splatting] and [Robotics], and then I merge the request. Thanks!

**Review by ruili3 — CHANGES_REQUESTED** [review]
Thanks for the update and the fixed typos!
I commented on L15 that the "\<br\>" tag can be removed so that there are no spaces between subfields. After that, I can merge the PR immediately!

File: README.md (hunk 1 above)
Comment (wzhi) [reply README.md]: Just removed the "\<br>" there!

**Review by ruili3 — APPROVED** [review]
thanks!

**Pull request merged**